pinc/wordcheck_engine.inc

```diff
@@ -860,7 +860,7 @@ function get_site_word_paths()
  *
  * @return object
  */
-function get_site_word_file($langcode3, $code)
+function get_site_word_file(string $langcode3, $code)
```

Comment:
```suggestion
function get_site_word_file(string $langcode3, string $code)
```

Reply:
FWIW `$code` was typed through an annotation but it's good to enforce it at runtime :smiley: